src/pages/documentation/mobile-core/lifecycle/tabs/index.md

```diff
@@ -1,9 +1,17 @@
-<Variant platform="android" task="add" repeat="2"/>
+<Variant platform="android" task="add" repeat="4"/>
 
-Import the library:
+Add the Lifecycle extension and it's dependency, the [Mobile Core](../index.md) extension to your project using the app's Gradle file.
 
 ```java
-   import com.adobe.marketing.mobile.*;
+implementation 'com.adobe.marketing.mobile:core:2.+'
+implementation 'com.adobe.marketing.mobile:lifecycle:2.+'
+```
+
```

Comment:
Are we also including the note about not using relative things here too?

Reply:
I missed adding that! Thanks for catching this 😄 . Added it now.